<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Локальный Мессенджер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
    <item>
     <widget class="QWidget" name="settings_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_nickname">
         <property name="text">
          <string>Ваш никнейм:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_nickname">
         <item>
          <widget class="QLineEdit" name="nicknameLineEdit">
           <property name="placeholderText">
            <string>Введите ник</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveNicknameButton">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Режим работы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="serverRadioButton">
            <property name="text">
             <string>Сервер</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="clientRadioButton">
            <property name="text">
             <string>Клиент</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_server">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="ipAddressLabel">
             <property name="text">
              <string>Ваш IP в сети: будет здесь</string>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="createButton">
             <property name="text">
              <string>Создать диалог</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_client">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_server_ip">
             <property name="text">
              <string>IP адрес сервера:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serverIpLineEdit">
             <property name="placeholderText">
              <string>Введите IP сервера</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Подключиться</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="chat_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="chatBrowser"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="messageLineEdit">
           <property name="placeholderText">
            <string>Введите ваше сообщение...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton">
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
